src/program/snabbnfv/snabb-nfvconfig.yang

```diff
@@ -0,0 +1,136 @@
+module snabb-nfvconfig {
+  namespace "http://snabb.co/nfvconfig";
+  prefix snabb-nfvconfig;
+
+  import ietf-yang-types { prefix yang; }
+  import ietf-inet-types { prefix inet; }
+
+  description
+    "This module describes the configuration of SnabbNFV.";
+
+
+  typedef pcap-filter {
+    type string;
+    description "A pcap-filter(7) expression.";
+  }
+
+  typedef gbps {
+    type decimal64 {
+      fraction-digits 4;
+      range "0..max";
+    }
+
+    description "Gigabytes per second.";
```

Comment:
This will be interpreted as Giga**bits** per second and not bytes. Why not use bps ?

```
typedef bps {
  type uint64;
  unit "bps";
  description "bits per second";
}
```